lambdasoup.opam

```
@@ -28,6 +28,7 @@ pin-depends: [
 
 build: [
   ["dune" "build" "-p" name "-j" jobs]
+  [make "test"] {with-test}
```

```suggestion
  ["dune" "runtest"] {with-test}
```

to avoid depending on `make`.